benchalerts/tests/unit_tests/mocked_responses/GET_conbench_compare_runs_some_baseline_some_contender.json

```diff
@@ -2,81 +2,117 @@
     "status_code": 200,
     "data": [
         {
-            "baseline": "0.03637 s",
-            "baseline_batch_id": "some-batch-uuid-1",
-            "baseline_error": null,
-            "baseline_id": "some-benchmark-uuid-1",
-            "baseline_run_id": "some-run-uuid-1",
-            "baseline_z_improvement": false,
-            "baseline_z_regression": false,
-            "baseline_z_score": null,
-            "batch": "file-read",
-            "benchmark": "snappy, nyctaxi_sample, parquet, arrow",
-            "change": "0.000%",
-            "contender": "0.03637 s",
-            "contender_batch_id": "some-batch-uuid-2",
-            "contender_error": null,
-            "contender_id": "some-benchmark-uuid-3",
-            "contender_run_id": "some-run-uuid-2",
-            "contender_z_improvement": false,
-            "contender_z_regression": true,
-            "contender_z_score": null,
-            "improvement": false,
-            "language": "Python",
+            "unit": "s",
             "less_is_better": true,
-            "regression": true,
-            "tags": {
-                "compression": "snappy",
-                "cpu_count": 2,
-                "dataset": "nyctaxi_sample",
-                "file_type": "parquet",
-                "input_type": "arrow",
-                "name": "file-read"
+            "baseline": {
+                "benchmark_name": "file-read",
+                "case_permutation": "snappy, nyctaxi_sample, parquet, arrow",
+                "language": "Python",
+                "value": "0.03637 s",
```

Comment:
I haven't gone in and debugged the failing `benchalerts` tests yet, but just noticed these numbers are still strings. You may need to re-do these mocks?

(And yes, I will set up actual integration tests with a CI Conbench API soon! 😬 )

Reply:
Oops, I edited most of them, but missed this one; fixed now